Cesium.Parser.Tests/PreprocessorTests/PreprocessorTests.cs

```diff
@@ -550,12 +550,27 @@ public Task IfExpressionCannotConsumeNonInteger()
 #endif
 ");
 
-    [Fact]
-    public Task IfExpressionLessOrEqualsWhenNotDefined() => DoTest(
+    [Fact, NoVerify]
+    public async Task IfExpressionLessOrEqualsWhenNotDefinedLeft()
+    {
+        var err = await Assert.ThrowsAsync<PreprocessorException>(async () => await DoTest(
 @"#if mycondition <= 1
 int foo() { return 0; }
 #endif
-");
+"));
+        Assert.Equal("The left-hand element of the expression was not found", err.Message);
+    }
+
+    [Fact, NoVerify]
+    public async Task IfExpressionLessOrEqualsWhenNotDefinedRight()
+    {
+        var err = await Assert.ThrowsAsync<PreprocessorException>(async () => await DoTest(
+@"#if 1 <= mycondition
+int foo() { return 0; }
+#endif
+"));
+        Assert.Equal("The right-hand element of the expression was not found", err.Message);
+    }
```

Comment:
I remembered that if it did not find a macro in the context, then it will be the true condition, I will correct it now